infra/packer/README.md

```diff
@@ -0,0 +1,46 @@
+# Umee Images
+
+## Public Docker Image Usage
+
+The latest (full) commit hash from the main branch can be used for the latest version of Umee.
+
+For example:
+```bash
+docker run -it us-docker.pkg.dev/umeedefi/stack/node:<full-git-sha>
```

Comment:
I'm assuming this won't work quite yet, until we merge this to main. Is that accurate?

Currently getting "manifest unknown: Failed to fetch" when I try the latest commits on main and on this branch, presumably just because the images don't exist yet.

i.e.

```
docker run -it us-docker.pkg.dev/umeedefi/stack/node:7eee4cc8bda457cfc8a27fae493e58273ec30b79

Unable to find image 'us-docker.pkg.dev/umeedefi/stack/node:7eee4cc8bda457cfc8a27fae493e58273ec30b79' locally
docker: Error response from daemon: manifest for us-docker.pkg.dev/umeedefi/stack/node:7eee4cc8bda457cfc8a27fae493e58273ec30b79 not found: manifest unknown: Failed to fetch "7eee4cc8bda457cfc8a27fae493e58273ec30b79".
See 'docker run --help'.


docker run -it us-docker.pkg.dev/umeedefi/stack/node:68858ed0d5d6d5ce568ce387a478eaaea6d448ec

Unable to find image 'us-docker.pkg.dev/umeedefi/stack/node:68858ed0d5d6d5ce568ce387a478eaaea6d448ec' locally
docker: Error response from daemon: manifest for us-docker.pkg.dev/umeedefi/stack/node:68858ed0d5d6d5ce568ce387a478eaaea6d448ec not found: manifest unknown: Failed to fetch "68858ed0d5d6d5ce568ce387a478eaaea6d448ec".
```

Reply:
Yes it is accurate that it won't track main at the moment.

Well. whats happening is that the pr workflows create a merge commit against the pr target and so the hash doesnt match the HEAD of the branch. PRs do this so that the CI is tested against a simulated main merge.

I can change the checkout settings to instead track the branch HEAD but that goes against the defaults/grain of prs. The merge commit proves that it would work one merging into main. Slightly inconvenient though I know because you have to look in the build log to find the image tag.

Example of snippet at end of build:
```
==> Builds finished. The artifacts of successful builds are:
--> umee-node.docker.final: Imported Docker image: sha256:100eb3d9811331cf19959f7e0a314dac3cbca96e7c79ed1d6f1b539780a98c35
--> umee-node.docker.final: Imported Docker image: us-docker.pkg.dev/umeedefi/stack/node:974c5645fb3275a14fd37f7a4e0649f2de4c0153 with tags us-docker.pkg.dev/umeedefi/stack/node:974c5645fb3275a14fd37f7a4e0649f2de4c0153
--> umee-node.docker.final: Imported Docker image: us-docker.pkg.dev/umeedefi/stack/node:974c5645fb3275a14fd37f7a4e0649f2de4c0153 with tags us-docker.pkg.dev/umeedefi/stack/node:974c5645fb3275a14fd37f7a4e0649f2de4c0153
```

Try this one based on a recent build:
`docker run -it us-docker.pkg.dev/umeedefi/stack/node:3acedf2fc8f2781b8148955f2866d1cf0ca39070`

I can make it match ci but the point of the merge